### DISCUSSION TITLE
When using dpdata to convert xyz format data of GAP potential, why does the number of converted frames differ too much from the declared frame number of about 18000?

### DISCUSSION THREAD

**Discussion by composites123** [deepmodeling/dpdata#584] [Q&A] [answered]
Hello, everyone:

I am trying to convert GAP20's xyz format training set using dpdata, but the output is only 109 frames, which is far from the declared 18,000 frames. What is the reason for this?

The commands I use are:
#################
import dpdata 
import numpy as np
data = dpdata.MultiSystems.from_file('Total.xyz',fmt='quip/gap/xyz') 
print('# the data contains %d frames' % len(data))
#################
Are there any mistakes?
The original data was too large to upload, so I uploaded some of it.
[Training_Set.zip](https://github.com/deepmodeling/dpdata/files/13520494/Training_Set.zip)

**Comment by njzjz** [comment, marked as answer]
`len(data)` is the number of systems. Use `data.get_nframes()` for the number of frames.

**Reply by composites123** [reply]
Thank you very much for your answer！
